src/css/app.sass

```diff
@@ -34,11 +34,11 @@ h3
 strong
   color: $secondary
 
-button
-  background: white
+button, a[type=button]
+  color: white !important
+  background-color: $secondary !important
```

Comment:
Do you have to do `!important`?
If possible, please avoid that.

Reply:
Although it's not pretty, it saves us many of the `color="secondary"` lines in `q-btn`. Without `!important`, `q-btn` would be transparent.

Reply:
I still think we can avoid that, But for now it is fine.